.buildkite/pipeline.yml

```diff
@@ -68,6 +68,18 @@ steps:
    commands: *run-nettest
    retry: *retry-nettest
    timeout_in_minutes: 120
+ - label: nettest-scheduled-delphinet
+   if: build.source == "schedule"
+   env:
+     TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER: "Y"
+     NETTEST_NODE_ADDR: "delphi.testnet.tezos.serokell.team"
+     NETTEST_NODE_PORT: "8732"
+     # Note that testnet moneybag can run out of tz. If this happened, someone should transfer it some
+     # more tz, its address: tz1ZLfAd3LVQUdj1vEX8YivD1B3sE23F29Fd
+     MONEYBAG: "unencrypted:edsk4Aq1Q3fc1ehMKTL55wcJxdz52r1iJggPHFkSFq57toaNNFVxKV"
```

Comment:
It may be a bit simpler to use the same address in both *nets (e. g. we won't have to write similar comments twice), but I don't insist on doing that.

Reply:
I mean you can transfer from `tz1ZLfAd3LVQUdj1vEX8YivD1B3sE23F29Fd` to `tz1ij8gUYbMRUXa4xX3mNvKguhaWG9GGbURn`.